sys/net/application_layer/dhcpv6/client.c

```diff
@@ -0,0 +1,932 @@
+/*
+ * Copyright (C) 2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ * @author  Martine Lenders <m.lenders@fu-berlin.de>
+ */
+
+#include <stdbool.h>
+
+#include "event.h"
+#include "log.h"
+#include "net/arp.h"
+#include "net/dhcpv6/client.h"
+#include "net/sock/udp.h"
+#include "random.h"
+#include "timex.h"
+#include "xtimer.h"
+#include "xtimer/implementation.h"
+
+#include "net/gnrc/ipv6/nib.h"
```

Comment:
the includes suggest heavy refactoring is needed...

Reply:
Well... someone is blocking #6413 for month now ;-).